src/site/apt/examples/deploy-signed-artifacts.apt

```diff
@@ -32,16 +32,51 @@ Deploy Signed Artifacts
 mvn deploy
 +----------+
 
- If you have configured this plugin according to the instructions in the
- {{{../usage.html}usage page}}, you just need to specify the passphrase for
- your private key on the command line like this:
+  If you have configured this plugin according to the instructions in the
+  {{{../usage.html}usage page}}, nothing changes for interactive sessions:
 
 +----------+
-mvn deploy -Dgpg.passphrase=thephrase
+mvn deploy
++----------+
+
+  And the gpg-agent will prompt you for passphrase.
+
+  If you use "batch" build (or build is invoked by Maven Release Plugin),
+  then gpg-agent cannot be used, hence supply passphrase via environment
+  variable:
+
++----------+
+MAVEN_GPG_PASSPHRASE=thephrase; mvn --batch-mode deploy
```

Comment:
The semicolon is not required. The command you have provided applies to Bourne shell and zsh, but will fail with csh and derivates.